docs/src/main/sphinx/connector/exasol.md

```diff
@@ -127,6 +131,32 @@ If no length is specified, the connector uses 2000000.
 Trino's `CHAR(n)` maps to `CHAR(n)` and vice versa if `n` is no greater than 2000.
 Exasol does not support longer values.
 
+(exasol-hashtype-mapping)=
+### Mapping `HASHTYPE` Types
+
+Exasol `HASHTYPE` columns are mapped to Trino's `VARBINARY` type (raw byte array)
+and vice versa, with the following rules and exceptions:
+
+- **Byte vs. bit definition**:  
+  `HASHTYPE(n BYTE)` and `HASHTYPE(m BIT)` are supported.
+    - `n` specifies the size in bytes (`1 ≤ n ≤ 1024`).
+    - `m` specifies the size in bits (`8 ≤ m ≤ 8192`, and must be a multiple of 8).  
+      Both definitions are equivalent (`m = n * 8`).
+
+- **Default length**:  
+  If neither `n` nor `m` is specified, Exasol defaults to `16 BYTE` (128-bit hash).
+
+- **Input format**:  
+  HASHTYPE values must be supplied as strings:
+    - Hexadecimal characters only.
+    - Dashes (`-`) are allowed in the string but ignored.
+    - Curly braces (`{}`) around the string are allowed.
+
+- **Validation**:
+    - The input string must match the expected size (`2n` hex characters).
+    - If the string length or characters are invalid, an exception is thrown.
+    - Any non-hex characters (other than `-` or `{}` / `}`) make the value invalid.
```

Comment:
This is implementation details. I don't think we want to document it in the connector.